src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1192,7 +1193,10 @@ private async Task<TokenValidationResult> ValidateTokenAsync(string token, JsonW
 
                         // Only try to re-validate using the newly obtained config if it doesn't reference equal the previously used configuration.
                         if (lastConfig != currentConfiguration)
-                            return decryptedJwt != null ? ValidateJWE(outerToken, decryptedJwt, validationParameters, currentConfiguration) : ValidateJWS(token, validationParameters, currentConfiguration); ;
+                        {
+                            validationParameters.ValidateIssuerWithLKG = false;
```

Comment:
Might need rename this flag to ValidateWithLKG since we might need this flag for [Design a signaling mechanism where LKG will signal the application layer when used.](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1923689)